diff --git a/launch/base_mighty.launch.py b/launch/base_mighty.launch.py
index 0000000..0000000 100644
--- a/launch/base_mighty.launch.py
+++ b/launch/base_mighty.launch.py
@@ -68,6 +68,7 @@ def generate_launch_description():
         # Map environment names to corresponding Gazebo world file names.
         world_mapping = {
             'simple_room': 'simple_room.world',
+            'fire_drill_room': 'fire_drill_room.world',
             'high_res_forest': 'big_forest_high_res.world',
             'static_uncertainty_test2': 'static_uncertainty_test2.world',
             'static_uncertainty_test3': 'static_uncertainty_test3.world',
