diff --git a/rviz/multi_mighty.rviz b/rviz/multi_mighty.rviz
index d894058..3fca52b 100644
--- a/rviz/multi_mighty.rviz
+++ b/rviz/multi_mighty.rviz
@@ -1562,6 +1562,13 @@ Visualization Manager:
         History Policy: Keep Last
         Reliability Policy: Reliable
         Value: /NX01/term_goal
+    - Class: rviz_default_plugins/SetGoal
+      Topic:
+        Depth: 5
+        Durability Policy: Volatile
+        History Policy: Keep Last
+        Reliability Policy: Reliable
+        Value: /NX02/term_goal
     - Class: rviz_default_plugins/PublishPoint
       Single click: true
       Topic:
