diff --git a/rviz/multi_mighty.rviz b/rviz/multi_mighty.rviz
index 19cb0a3..aafc5dd 100644
--- a/rviz/multi_mighty.rviz
+++ b/rviz/multi_mighty.rviz
@@ -31,6 +31,200 @@ Panels:
 Visualization Manager:
   Class: ""
   Displays:
+    - Alpha: 1
+      Autocompute Intensity Bounds: true
+      Autocompute Value Bounds:
+        Max Value: 10
+        Min Value: -10
+        Value: true
+      Axis: Z
+      Channel Name: intensity
+      Class: rviz_default_plugins/PointCloud2
+      Color: 239; 41; 41
+      Color Transformer: FlatColor
+      Decay Time: 0
+      Enabled: false
+      Invert Rainbow: false
+      Max Color: 255; 255; 255
+      Max Intensity: 4096
+      Min Color: 0; 0; 0
+      Min Intensity: 0
+      Name: NX01 EGO occupancy_inflate
+      Position Transformer: XYZ
+      Selectable: true
+      Size (Pixels): 3
+      Size (m): 0.10000000149011612
+      Style: Flat Squares
+      Topic:
+        Depth: 5
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX01/grid_map/occupancy_inflate
+      Value: false
+    - Alpha: 1
+      Autocompute Intensity Bounds: true
+      Autocompute Value Bounds:
+        Max Value: 10
+        Min Value: -10
+        Value: true
+      Axis: Z
+      Channel Name: intensity
+      Class: rviz_default_plugins/PointCloud2
+      Color: 239; 41; 41
+      Color Transformer: FlatColor
+      Decay Time: 0
+      Enabled: false
+      Invert Rainbow: false
+      Max Color: 255; 255; 255
+      Max Intensity: 4096
+      Min Color: 0; 0; 0
+      Min Intensity: 0
+      Name: NX02 EGO occupancy_inflate
+      Position Transformer: XYZ
+      Selectable: true
+      Size (Pixels): 3
+      Size (m): 0.10000000149011612
+      Style: Flat Squares
+      Topic:
+        Depth: 5
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX02/grid_map/occupancy_inflate
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX01 EGO goal_point
+      Namespaces:
+        {}
+      Topic:
+        Depth: 2
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX01/goal_point
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX01 EGO global_list
+      Namespaces:
+        {}
+      Topic:
+        Depth: 2
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX01/global_list
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX01 EGO init_list
+      Namespaces:
+        {}
+      Topic:
+        Depth: 2
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX01/init_list
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX01 EGO optimal_list
+      Namespaces:
+        {}
+      Topic:
+        Depth: 2
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX01/optimal_list
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX01 EGO a_star_list
+      Namespaces:
+        {}
+      Topic:
+        Depth: 20
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX01/a_star_list
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX02 EGO goal_point
+      Namespaces:
+        {}
+      Topic:
+        Depth: 2
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX02/goal_point
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX02 EGO global_list
+      Namespaces:
+        {}
+      Topic:
+        Depth: 2
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX02/global_list
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX02 EGO init_list
+      Namespaces:
+        {}
+      Topic:
+        Depth: 2
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX02/init_list
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX02 EGO optimal_list
+      Namespaces:
+        {}
+      Topic:
+        Depth: 2
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX02/optimal_list
+      Value: false
+    - Class: rviz_default_plugins/Marker
+      Enabled: false
+      Name: NX02 EGO a_star_list
+      Namespaces:
+        {}
+      Topic:
+        Depth: 20
+        Durability Policy: Volatile
+        Filter size: 10
+        History Policy: Keep Last
+        Reliability Policy: Best Effort
+        Value: /NX02/a_star_list
+      Value: false
     - Class: rviz_common/Group
       Displays:
         - Alpha: 1
